timetabler/ssc/course.py

```diff
@@ -1,3 +1,6 @@
+import re
+
+
 class Course(object):
     def __init__(self, dept, number,
                  lectures=None, labs=None, tutorials=None, discussions=None):
```

Comment:
Will do. Is there some global/container it can be set to or must it be passed through?